AC3.2 Unit 2 Final Assessment Project/AC3.2 Unit 2 Final Assessment Project/CrayonTableViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  CrayonTableViewController.swift
+//  AC3.2 Unit 2 Final Assessment Project
+//
+//  Created by Edward Anchundia on 10/6/16.
+//  Copyright © 2016 Edward Anchundia. All rights reserved.
+//
+
+import UIKit
+
+class CrayonTableViewController: UITableViewController {
+
+    internal var crayons: [Crayon] = []
+    let cellIdentifier: String = "CrayonTableViewCell"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+
+        for c in crayolaColors {
+            if let crayon = Crayon(fromDict: c) {
+                crayons.append(crayon)
+            }
+        }
+        
+        dump(crayons)
+        
+    }
+    
+
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return crayons.count
+    }
+
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath)
+
+        // Configure the cell...
+        cell.textLabel?.text = crayons[indexPath.row].name
```

Comment:
Easier to pass crayon to variable like:
`let crayon = crayons[indexPath.row]`

then you can assign cell.backgroundColor like:
`cell.backgroundColor = UIColor(colorLiteralRed: Float(crayon.red), green: Float(crayon.green), blue: Float(crayon.blue), alpha: 1.0)`